cluster/images/provider-tf-aws-controller/Dockerfile

```diff
@@ -4,8 +4,30 @@ RUN apk --no-cache add ca-certificates bash
 ARG ARCH
 ARG TINI_VERSION
 
+ENV USER_ID=1001
+
+# Setup Terraform environment
+ENV TERRAFORM_VERSION 1.0.5
+ENV TERRAFORM_PROVIDER_AWS_VERSION 3.56.0
+ENV PLUGIN_DIR /terraform/provider-mirror/registry.terraform.io/hashicorp/aws/${TERRAFORM_PROVIDER_AWS_VERSION}/linux_${ARCH}
+ENV TF_CLI_CONFIG_FILE /terraform/.terraformrc
+ENV TF_FORK 0
+
+RUN mkdir -p ${PLUGIN_DIR}
+
+ADD https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_${ARCH}.zip /tmp
+ADD https://releases.hashicorp.com/terraform-provider-aws/${TERRAFORM_PROVIDER_AWS_VERSION}/terraform-provider-aws_${TERRAFORM_PROVIDER_AWS_VERSION}_linux_${ARCH}.zip /tmp
+ADD terraformrc.hcl ${TF_CLI_CONFIG_FILE}
+
+RUN unzip /tmp/terraform_${TERRAFORM_VERSION}_linux_${ARCH}.zip -d /usr/local/bin \
+  && rm /tmp/terraform_${TERRAFORM_VERSION}_linux_${ARCH}.zip \
+  && unzip /tmp/terraform-provider-aws_${TERRAFORM_PROVIDER_AWS_VERSION}_linux_${ARCH}.zip -d ${PLUGIN_DIR} \
```

Comment:
nit: If you would like, we may explicitly give execute permissions (`chmod`) for the plugin binary.

Reply:
Done, thanks!